src/components/m-button/index.js

```diff
@@ -4,8 +4,7 @@ import BaseComponentGlobal from '../../js/abstract/base-component-global';
 import wcdomready from '../../js/wcdomready';
 
 class AXAButton extends BaseComponentGlobal {
-  static get observedAttributes() { return ['arrow', 'classes', 'color', 'ghost', 'motion', 'size', 'tag', 'url']; }
-
+  static get observedAttributes() { return ['arrow', 'classes', 'color', 'ghost', 'motion', 'size', 'tag', 'url', 'icon', 'reverse']; }
```

Comment:
I just realised, if it's an `<a/>` tag we would need `href`, `lang`, `target`, etc. (I would replace url with href to be consistent)
And in case of `<button/>` tag all valid button attributes `type`, etc.